script.js

```diff
@@ -49,4 +49,110 @@ iconToastDemo.addEventListener('click', () => {
     setTimeout(() => toggleShowClass(iconToast), 3000);
 });
 
-iconToastClose.addEventListener('click', () => toggleShowClass(iconToast))
\ No newline at end of file
+iconToastClose.addEventListener('click', () => toggleShowClass(iconToast))
+
+// ----------------slider-----------------------
+
+const rangeInput = document.querySelectorAll(".range"),
+    rangeInputField = document.querySelectorAll(".range-input-field"),
+    range = document.querySelector(".slider .progress");
+let priceGap = 1000;
+
+rangeInputField.forEach(input => {
+    input.addEventListener("input", e => {
+        let minPrice = parseInt(rangeInputField[0].value);
+        let maxPrice = parseInt(rangeInputField[1].value);
+
+        if ((maxPrice - minPrice >= priceGap) && maxPrice <= rangeInput[1].max) {
+            if (e.target.className === "min-field") {
+                rangeInput[0].value = minPrice;
+                range.style.left = ((minPrice / rangeInput[0].max) * 100) + "%";
+            } else {
+                rangeInput[1].value = maxPrice;
+                range.style.right = 100 - (maxPrice / rangeInput[1].max) * 100 + "%";
+            }
+        }
+    });
+});
+rangeInput.forEach(input => {
+    input.addEventListener("input", e => {
+        let minVal = parseInt(rangeInput[0].value),
+            maxVal = parseInt(rangeInput[1].value);
+        if ((maxVal - minVal) < priceGap) {
+            if (e.target.className === "min-range") {
+                rangeInput[0].value = maxVal - priceGap
+            } else {
+                rangeInput[1].value = minVal + priceGap;
+            }
+        } else {
+            rangeInputField[0].value = minVal;
+            rangeInputField[1].value = maxVal;
+            range.style.left = ((minVal / rangeInput[0].max) * 100) + "%";
+            range.style.right = 100 - (maxVal / rangeInput[1].max) * 100 + "%";
+        }
+    });
+});
+
+// -----------------------carousel------------------
+var slidePosition = 1;
+CarouselSlideShow(slidePosition);
+
+function moveSlide(n) {
+    CarouselSlideShow(slidePosition += n);
+}
+
+function currentSlide(n) {
+    CarouselSlideShow(slidePosition = n);
+}
+
+function CarouselSlideShow(n) {
+    var i;
+    var slides = document.querySelectorAll(".container");
+    var circles = document.querySelectorAll(".dots");
```

Comment:
Here also you can use const or let.

Reply:
I tried using const and let, but it gave scope errors. That's why I used var. I will fix this once all of my projects are completed. I hope that's okay.

Reply:
sure.

Reply:
I have changed the declarations. Thanks